catalog/internal/catalog/genqlient/README.md

```diff
@@ -0,0 +1,30 @@
+## Using Genqlient with the Red Hat Ecosystem Catalog
```

Comment:
would it make sense to add a sentence in this readme ~like

This represent also an example implementation of a CatalogSource that retrieves the metadata for the Model Catalog based on GraphQL queries on an external system